spvc/.clang-format

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This should live in the root dir and all the source using the same formatting. It looks like shaderc isn't formatted yet?

Reply:
shaderc has .clang-format files in glslc, libshaderc, libshaderc_spvc and libshaderc_util.  They are all the same.
I'll make a PR to have one at the root.